static/css/ux-library.css

```diff
@@ -0,0 +1,584 @@
+/*Fonts*/
+p{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    letter-spacing: 0.03px;
+}
+
+h1{
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
In the design document, the primary font for h1 is Helvetica not Arial

Reply:
Switched fonts around so Helvetica is first